src/DataBuilderInterface.php

```diff
@@ -2,7 +2,10 @@
 
 namespace Rollbar;
 
+use Rollbar\Payload\Data;
+use Throwable;
+
 interface DataBuilderInterface
 {
-    public function makeData($level, $toLog, $context);
+    public function makeData(string $level, Throwable|string $toLog, array $context): Data;
```

Comment:
assuming "level" implies a Rollbar log level, can it be modeled as an enum type (if PHP supports such types)?

Reply:
@akornich - Interestingly, that model already exists: [`Rollbar\Payload\Level`][1]. API on the perimeter explicitly accept [`Level|string`][2], while internal "friend" API do so implicitly. [I commented][3] some while ago that this implicit type allowance isn't ideal, and that internally we should stick with `Level`.

[1]:https://github.com/rollbar/rollbar-php/blob/master/src/Payload/Level.php
[2]:https://github.com/rollbar/rollbar-php/blob/master/src/RollbarLogger.php#L92
[3]:https://github.com/rollbar/rollbar-php/blob/master/src/RollbarLogger.php#L99-L103